pkg/accountability/simpleacc/accountability.go

```diff
@@ -0,0 +1,137 @@
+package simpleacc
+
+import (
+	"github.com/filecoin-project/mir/pkg/accountability/simpleacc/common"
+	"github.com/filecoin-project/mir/pkg/accountability/simpleacc/internal/certificates/fullcertificates"
+	incommon "github.com/filecoin-project/mir/pkg/accountability/simpleacc/internal/common"
+	"github.com/filecoin-project/mir/pkg/accountability/simpleacc/internal/poms"
+	"github.com/filecoin-project/mir/pkg/accountability/simpleacc/internal/predecisions"
+	"github.com/filecoin-project/mir/pkg/dsl"
+	"github.com/filecoin-project/mir/pkg/factorymodule"
+	"github.com/filecoin-project/mir/pkg/logging"
+	"github.com/filecoin-project/mir/pkg/modules"
+	accpbtypes "github.com/filecoin-project/mir/pkg/pb/accountabilitypb/types"
+	factorypbtypes "github.com/filecoin-project/mir/pkg/pb/factorypb/types"
+	t "github.com/filecoin-project/mir/pkg/types"
+)
+
+// ModuleConfig sets the module ids. All replicas are expected to use identical module configurations.
+type ModuleConfig = common.ModuleConfig
+
+// ModuleParams sets the values for the parameters of an instance of the protocol.
+// All replicas are expected to use identical module parameters.
+type ModuleParams = common.ModuleParams
+
+// NewModule creates a new instance of the (optinal) accountability
+// module.
+// This module can receive predecisions from an
+// ordering module (instead of the ordering module delivering them to the
+// application layer directly). It performs two all-to-all broadcasts
+// with signatures to ensure accountability. The first broadcast is a
+// signed predecision per participant. In the second broadcast, each
+// participant broadcasts a certificate containing a strong quorum of
+// signed predecisions that they each delivered from the first
+// broadcast. Termination occurs once a process receives a strong quorum
+// of correctly signed predecisions.
+// *Accountability* states that if an adversary (controlling less than a
+// strong quorum, but perhaps more or as much as a weak quorum) causes
+// a disagreement (two different correct processes delivering different
+// decisions) then there is at least a weak quorum of processes for which
+// all correct processes eventually receive Proofs-of-Misbehavior (PoMs).
```

Comment:
I changed the sentence so that it is easier to understand:
```

// ...a disagreement (two different correct processes delivering different
// decisions) then all correct processes eventually receive Proofs-of-Misbehavior (PoMs)
// for a provably malicious coalition at least the size of a weak quorum.
```